pelops/features/hog.py

```diff
@@ -20,14 +21,38 @@ def produce_features(self, chip):
         img = self.get_image(chip)
         img = img.resize(self.image_size, Image.BICUBIC)
         img_x, img_y = img.size
+
+        # Calculate histogram of each channel
```

Comment:
# Add a histogram for up to the first 3 channels
# missing channels will be replaced with -1
# only the first 3 channels matter, all other channels will be ignored